swim/disseminator_test.go

```diff
@@ -119,7 +119,7 @@ func (s *DisseminatorTestSuite) TestFullSync() {
 		s.m.MakeAlive(address, s.incarnation)
 	}
 
-	changes := s.d.FullSync()
+	changes := s.d.MembershipAsChanges()
```

Comment:
Lets change the name of this test according to the rename of the `FullSync` function